### DISCUSSION TITLE
Installing a package for GLPI 10.0.2 version

### DISCUSSION THREAD

**Discussion by ksilva-kwn** [community/community#132400] [Programming Help]
### Select Topic Area

Question

### Body

Great day guys

I'm trying to install a mandatory GLPi package (in version 10.0.2) on my environment, on Oracle Linux 8, but it presents an error when I install the **GD** package.

![image](https://github.com/user-attachments/assets/36404b78-d131-4056-a536-4d766cb089d7)

Obs*

My php version is 8.0.30

Can anyone help me solve this problem?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @ksilva-kwn ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by ksilva-kwn** [reply]
Thanks bro!!!

**Comment by HemalathaT2** [comment]
The error message indicates that the php80-php-gd package requires libgd.so.103()(64bit), but none of the available providers can install it. This is because the required libraries (libraqm, libimagequant, and libavif) are missing
Determine the specific packages that provide the missing libraries (libraqm, libimagequant, and libavif) for your Oracle Linux 8 environment. You can use tools like yum search or the official Oracle Linux repository to find them
